routes/movies-routes.js

```diff
@@ -0,0 +1,79 @@
+// starter code in both routes/celebrities.routes.js and routes/movies.routes.js
+const router = require("express").Router();
+
```

Comment:
```suggestion
const Celebrity = require("./../models/Celebrity.model")
```
La requerimos aquí, ahora entenderás porque.